src/pages/workspace/WorkspaceNewRoomPage.js

```diff
@@ -114,7 +114,7 @@ class WorkspaceNewRoomPage extends React.Component {
         // Workspaces are policies with type === 'free'
         const workspaceOptions = _.map(
             _.filter(this.props.policies, policy => policy && policy.type === CONST.POLICY.TYPE.FREE),
-            policy => ({label: policy.name, key: policy.id, value: policy.id}),
+            policy => ({label: policy.name.replace(/ /g, '\u00a0'), key: policy.id, value: policy.id}),
```

Comment:
@Santhosh-Sellavel 

> Bug: Missed on New room flow

Thanks for pointing this out. The root cause is we are using the option tag in the picker component (react-native-picker-select --> react-native-picker --> @react-native-picker/picker --> option tag) and the option tag will automatically remove white space


<img width="1440" alt="Screen Shot 2023-03-10 at 13 38 24" src="https://user-images.githubusercontent.com/113963320/224242082-a611f622-d598-4dbf-8e0b-6c9ac8f772c6.png">

So we need to encode the white space into the &nbsp then the white space will be displayed correctly in the option tag like this

<img width="1440" alt="Screen Shot 2023-03-10 at 13 43 03" src="https://user-images.githubusercontent.com/113963320/224242901-077c7ed4-8d23-4033-a758-55aab52266e9.png">
<img width="1440" alt="Screen Shot 2023-03-10 at 13 43 32" src="https://user-images.githubusercontent.com/113963320/224243044-e30f98d6-e29d-40e0-8820-6356a2e1111b.png">

So I think this change is good. This is result 
<img width="1440" alt="Screen Shot 2023-03-10 at 13 45 24" src="https://user-images.githubusercontent.com/113963320/224243449-439ef975-e152-4a22-9532-9af382007443.png">
<img width="1440" alt="Screen Shot 2023-03-10 at 13 45 32" src="https://user-images.githubusercontent.com/113963320/224243456-7d8e21ec-4796-42f8-bde9-20ff96ca7b6c.png">



Please tell me about your thought

Reply:
Hi @Santhosh-Sellavel Could you help to take a look at my comment?

Reply:
This is kind of weird and hacky. Maybe should be handled separately Thoughts @alex-mechler?

Reply:
I agree, this feels very hacky. Lets note this as a separate issue, and not handle it here.

Reply:
@Santhosh-Sellavel I just reverted the hacky fix. Please check again and tell me there are any problem

Reply:
Hi @Santhosh-Sellavel Could you take a look at this PR? Thanks

Reply:
I created a follow up issue for this. https://github.com/Expensify/App/issues/16329